StyleCop.Analyzers/StyleCop.Analyzers/OrderingRules/SA1206DeclarationKeywordsMustFollowOrder.cs

```diff
@@ -111,38 +86,39 @@ private static void HandleDeclaration(SyntaxNodeAnalysisContext context)
             CheckModifiersOrderAndReportDiagnostics(context, modifiers);
         }
 
+        // TODO: Can this be merged with DeclarationModifiersHelper.GetModifiersFromDeclaration?
```

Comment:
I think `DeclarationModifiersHelper.GetModifiers` should work in places where this method is used.